src/Perl6/SysConfig.nqp

```diff
@@ -2,8 +2,14 @@ class Perl6::SysConfig is HLL::SysConfig {
     has %!rakudo-build-config;
     has $!rakudo-home;
 
-    method BUILD() {
-        %!rakudo-build-config := nqp::hash();
+    method new(:%rakudo-build-config) {
+        my $obj := nqp::create(self);
+        $obj.BUILD(:%rakudo-build-config);
+        $obj
+    }
+
+    method BUILD(:%rakudo-build-config) {
+        %!rakudo-build-config := %rakudo-build-config;
```

Comment:
Is this a valid and good way to construct a subclass with a named parameter in NQP?
A small test shows, that the superclass (HLL::SysConfig) is actually called, but I don't know how.

Reply:
Update: It seems, that the super class constructor is called, but the subclass then starts with a blank superclass. So this actually doesn't work.